app/src/main/java/org/mozilla/focus/fragment/onboarding/OnboardingSecondScreenCompose.kt

```diff
@@ -0,0 +1,193 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.fragment.onboarding
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.IconButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import mozilla.components.ui.colors.PhotonColors
+import org.mozilla.focus.R
+import org.mozilla.focus.ui.theme.FocusTheme
+import org.mozilla.focus.ui.theme.focusTypography
+
+@Composable
+@Preview
+private fun OnBoardingSecondScreenComposePreview() {
+    FocusTheme {
+        OnBoardingSecondScreenCompose({}, {}, {})
+    }
+}
+
+/**
+ * Displays the second onBoarding screen
+ *
+ * @param setAsDefaultBrowser when the user clicks on SetDefaultBrowser button
+ * @param skipScreen when the user clicks on skip Button
+ * @param onCloseButtonClick the user clicks on close onBoarding flow
+ */
+@Composable
+@Suppress("LongMethod")
+fun OnBoardingSecondScreenCompose(
+    setAsDefaultBrowser: () -> Unit,
+    skipScreen: () -> Unit,
+    onCloseButtonClick: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .fillMaxHeight()
+            .background(
+                brush = Brush.linearGradient(
+                    colors = listOf(
+                        colorResource(R.color.biometric_background_fragment_one),
```

Comment:
Same question as above. Do we need to extend this scope of this color resource by renaming the file or duplicate it to keep specialized usage?

Reply:
Done